roles/cockpit_run_verify/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+# vim: set ft=ansible:
+#
+- name: Set cockpit container name for RHELAH
+  set_fact:
+    cockpit_cname: "registry.access.redhat.com/rhel7/cockpit-ws"
+  when: ansible_distribution == "RedHat"
+
+- name: Set cockpit container name for Fedora/CentOS
+  set_fact:
+    cockpit_cname: "cockpit/ws"
+  when: ('CentOS' in ansible_distribution) or ansible_distribution == "Fedora"
+
+- name: Run cockpit container
+  command: atomic run {{ cockpit_cname }}
+
+- name: Verify port 9090 is open
+  wait_for:
+    port: 9090
+    timeout: 120
```

Comment:
It should start up right away.  Do you think 30 seconds would be good before it fails?